### PR TITLE
chore: [docs] Update docs to install ee-helm-chartv2

### PR DESCRIPTION
**Pull request by pratapaprasanna**
This PR adds three documents.

1. A document detailing the installation of Appsmith Business Edition, from scratch, using the new Helm charts at `helm-ee.appsmith.com`. This is `installation-guides/kubernetes/business-edition.md`.

2. A document on upgrading an Appsmith Community Edition instance (which would be using `helm.appsmith.com`), to Appsmith Business Edition, with `helm-ee.appsmith.com`. This is `upgrade-to-business-edition/kubernetes.md`.

3. A document on migrating an Appsmith Business Edition instance, that's running with `helm.appsmith.com`, to start using the new EE-specific Helm chart, at `helm-ee.appsmith.com`. This is `installation-guides/kubernetes/migrate-to-ee-chart.md`.

### CODE REVIEW

File: website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md
Comment (jnikhila) [inline website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md:5]: @danciaclara and @sharat87 - this file is ready for review. Please review and add your comments.

File: website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md
Comment (sharat87) [inline website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md:3]: This sentence is off. It reads like "move from Business Edition ... to Business Edition".

This doc, is for those who are using Appsmith Business Edition, with the old Helm chart at `helm.appsmith.com`, to move to the new helm chart, at `helm-ee.appsmith.com`. Can we rephrase to reflect that please?

File: website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md
Comment (sharat87) [inline website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md:5]: Are you sure of the wording here? We usually _export data_, or _take a backup_, but _exporting backup_ sounds a bit off to me. Either way is fine though, just wanted to point out.

File: website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md
Comment (sharat87) [inline website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md:25]: Can we undo the change here as well please? This reads like it's just an instruction on _how_ to do something. Not like an instruction to just go do it. We're not informing them on how to exit the shell, we're telling them to exit the shell. Something like `Please exit the shell, by running the following command:` or something.

I see a lot of these wordings, can we change them please?

File: website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md
Comment (sharat87) [inline website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md:47]: Can we keep the `"` characters please? Including them is recommended, and with this it's not immediately obvious that they have to wrap the values in double-quotes.

File: website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md
Comment (sharat87) [inline website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md:50]: Keycloak, is the name of a product, just like Appsmith, please capitalize, unless it's part of a command. 🙂

File: website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md
Comment (sharat87) [inline website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md:124]: I'd request undoing this change as well. The given link is not _more_ information, it's _the_ information. It's not optional, like, _see this link if you want more information_. It's more like, _follow instructions from this link_, if I'm making sense. 😅

File: website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md
Comment (sharat87) [inline website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md]: Can we start including a conclusion/final-state statement like this on our docs? Otherwise it feels we've stopped midway and left the readers hanging.

**Review by sharat87 — COMMENTED** [review]
Can we change the voicing that reads `To do X, please run this:` to actually asking to run the command please? Currently, it reads like _to do X, please do Y_, but that leaves the question of whether the reader should actually do Y, or if it's optional. Can we just say "please do Y", or "please do Y, which will X", or "please do Y, which will achieve X", or something? What say?

File: website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md
Comment (sharat87) [reply website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md:2]: Hey, yes, the doc is similar, but I'd suggest keeping it separate.

I've already included the title that I think would be right for this, or, how about "Migrate to Business Edition Helm Chart"...? Not really sure beyond that.

Comment (danciaclara) [reply website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md:2]: Just rearranging the words a bit - **Migrate to Helm Chart (BE)**

File: website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md
Comment (danciaclara) [inline website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md:137]:
```suggestion
## Install new Helm chart 
```

File: website/docs/getting-started/setup/upgrade-to-business-edition/README.md
Comment (danciaclara) [inline website/docs/getting-started/setup/upgrade-to-business-edition/README.md:23]: logo appears to be broken
Screenshot 2022-12-19 at 17 40 26

File: website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md
Comment (danciaclara) [inline website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md:2]:
```suggestion
description: Follow the guide to upgrade the Appsmith Community Edition installation on Kubernetes to Business Edition.
```

File: website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md
Comment (danciaclara) [inline website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md:7]:
```suggestion
The Business Edition (BE) Helm chart installation includes support for Horizontal Pod Auto Scaling (HPA), which allows Appsmith pods to scale automatically based on the current load. Additionally, this means that Appsmith pods are managed using a Kubernetes _deployment_ resource instead of a _stateful-set_ resource. Follow this guide to upgrade from the Community Edition (CE) to the Business Edition (BE) installed on Kubernetes.
```

File: website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md
Comment (danciaclara) [inline website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md:9]:
```suggestion
## Backup data
```

File: website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md
Comment (danciaclara) [inline website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md:26]: remove path from code block

File: website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md
Comment (danciaclara) [inline website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md:41]:
```suggestion
   Set the parameters in the `applicationConfig` section in the values.yaml file
```

File: website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md
Comment (danciaclara) [inline website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md:48]:
```suggestion
## Uninstall CE Helm chart
```

File: website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md
Comment (danciaclara) [inline website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md:84]:
```suggestion
3. Choose or [create a shared file system.](/getting-started/setup/installation-guides/kubernetes/business-edition#create-a-shared-file-system).
```

File: website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md
Comment (danciaclara) [inline website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md:97]:
```suggestion
## Install EE Helm chart 
```

File: website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md
Comment (danciaclara) [inline website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md:140]:
```suggestion
   Monitor the output as shown below:
```

File: website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md
Comment (danciaclara) [inline website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md:151]:
```suggestion
   When you see the output, as shown above, press <kbd>ctrl+c</kbd> to stop.
```

File: website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md
Comment (danciaclara) [inline website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md:142]: This should ideally be a screenshot. If we can disable the copy button for such output-related content, that would be great.

File: website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md
Comment (danciaclara) [inline website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md:159]: Let this be under the subheading 'Troubleshooting'

**Review by danciaclara — CHANGES_REQUESTED** [review]

File: website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md
Comment (jnikhila) [reply website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md:3]: I think adding new and old installations doesn't sound good. How about:

> Use the following guide to move to the Business Edition Helm chart v2 (available at `helm-ee.appsmith.com`). This version includes Horizontal Pod Auto Scaling (HPA) capability, which enables Appsmith pods to scale automatically based on the current workload.

File: website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md
Comment (jnikhila) [reply website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md:5]: We are using 'Export data' everywhere in the docs for taking backups.

File: website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md
Comment (jnikhila) [reply website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md:22]: We will change these all once the component available. For now, I've kept it as is.

File: website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md
Comment (jnikhila) [reply website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md:25]: We don't use words like please, should, could, would, and so on in docs. What do you think if I reword it as below:

> Run the below command to exit the shell.

File: website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md
Comment (jnikhila) [reply website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md:50]: To maintain the consistency, I'll keep it as `Export Keycloak backup.`

File: website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md
Comment (jnikhila) [reply website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md:124]: I thought users would get more information on this. So added it like that.
In that case, I think it will be back and forth for users to navigate to another page, and then perform the action and move back to this page to proceed.  I guess embedding that info here would make more sense. CC: @danciaclara - Your thoughts?

File: website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md
Comment (jnikhila) [reply website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md:9]: As we have Export backup in docs to refer to Backup data, I kept it in this doc too.

File: website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md
Comment (jnikhila) [reply website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md:26]: These changes will happen once we have the component merged.

File: website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md
Comment (jnikhila) [reply website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md:84]: Again I guess this step is needed as mandatory, so should we add it here. @sharat87 raised it for migrate to v2 helm chart BE.

File: website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md
Comment (jnikhila) [reply website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md:142]: As it's easy for users to copy and search the text in the terminal. IMO we should keep it as text. We'll change the highlighting once we have the component merged.

File: website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md
Comment (jnikhila) [reply website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md:137]: Let's discuss this.

File: website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md
Comment (jnikhila) [reply website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md:2]: @sharat87 - Your thoughts?

File: website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md
Comment (jnikhila) [reply website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md:124]: As discussed, embed the section 'create a shared file system' here.

File: website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md
Comment (jnikhila) [reply website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md]: Can you suggest how a user can verify the new installation?

File: website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx
Comment (jnikhila) [inline website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx:22]: @danciaclara, @sharat87 and @pratapaprasanna - The Business Edition instructions are updated and ready for review.

File: website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md
Comment (danciaclara) [reply website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md]: I think it is important to mention from where they are moving too.  You can modify it a bit as below

> Follow this guide to move the Appsmith Business Edition installation from the Helm chart at `helm.appsmith.com` to the new Helm chart at `helm-ee.appsmith.com`. This installation supports Horizontal Pod Auto Scaling (HPA), which enables Appsmith pods to scale automatically based on the current load.

File: website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md
Comment (danciaclara) [reply website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md]: Let's use the verb first

File: website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md
Comment (danciaclara) [reply website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md]: Apologies, I take this back :) As per the Google style guide file names do appear in code blocks   - https://developers.google.com/style/code-in-text#some-specific-items-to-put-in-code-font.  The Docusaurus inline code blocks looks a bit weird too. But we had a discussion around this and Nikhila is changing the CSS a bit to match the inline code style in Google style guide.

Let's also follow the above guide for when and when not to use things in code blocks.

File: website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md
Comment (danciaclara) [reply website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md]: Let's leave it as per the google style guide as shared in above comment

File: website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md
Comment (danciaclara) [reply website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md]: @jnikhila I thought about this again and the old looks odd there. I'll let you to decide on this

File: website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md
Comment (danciaclara) [reply website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md]: I saw the new change. Keep it as "Install Helm Chart v2"

File: website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx
Comment (danciaclara) [reply website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx:22]: hey I'm not able to find my comments for that page before it was reworked and moved. Will that be lost?

File: website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md
Comment (danciaclara) [reply website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md:142]: As mentioned above, let's keep it as is. I was wrong :)

File: website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md
Comment (danciaclara) [reply website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md]: yes let's add that step here itself so that users do not have to migrate

File: website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md
Comment (danciaclara) [reply website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md:26]: ignore this comment. Closing conversation

File: website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx
Comment (danciaclara) [reply website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx:22]: I'm a bit worried that the original Configure and Update section for CE got hidden in the tab now. Should we keep this as a separate page itself?

Screenshot 2022-12-24 at 02 59 40
Screenshot 2022-12-24 at 03 00 01

**Review by sharat87 — APPROVED** [review]

File: website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx
Comment (jnikhila) [reply website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx:22]: I've hidden the sections as the TOC was cluttered and would have confused users.

**Pull request merged**
